orchestra/cmds/update.py

```diff
@@ -96,20 +95,29 @@ def pull_binary_archive(name, config):
 def clone_binary_archive(name, url, config):
     logger.info(f"Trying to clone binary archive from remote {name} ({url})")
     binary_archive_path = os.path.join(config.binary_archives_dir, name)
-    env = dict(os.environ)
+    env = os.environ.copy()
     env["GIT_SSH_COMMAND"] = "ssh -oControlPath=~/.ssh/ssh-mux-%r@%h:%p -oControlMaster=auto -o ControlPersist=10"
     env["GIT_LFS_SKIP_SMUDGE"] = "1"
-    result = subprocess.run(["git", "clone", url, binary_archive_path], env=env)
+    result = run_subprocess(
+        ["git", "clone", url, binary_archive_path],
+        environment=env,
+        loglevel="DEBUG",
+        check_returncode=False
+    )
```

Comment:
`run_script_internal`? `run_subprocess_internal`?
We really need to cut down direct usages of `run_script`/`run_subprocess`.